csrc/root_domain_map.cpp

```diff
@@ -121,6 +121,37 @@ std::unordered_map<IterDomain*, IterDomain*> PairwiseRootDomainMap::map(
   const auto producer_root =
       TensorDomain::noReductions(producer->maybeRFactor());
   const auto& consumer_root = consumer->root();
+
+  // Map MatmulOp extents in reverse
```

Comment:
Please place a description of the logic in a comment here. You may want to refer to https://pytorch.org/docs/stable/generated/torch.matmul.html. Ideally a reader should be able to quickly read this and confirm that it implements the torch.matmul interface.

Reply:
Refactored the mapping logic to a separate function. Let me know if the new comment is clear about the functionality